### DISCUSSION TITLE
How do i  migrate the files under /api/files

### DISCUSSION THREAD

**Discussion by MalikWhitten67** [pockethost/pockethost#186] [Q&A] [answered]
I wanted to locally run postr api from home to stress test, but I cannot get the files that are from posts to work.

**Comment by benallfree** [comment]
Do you mean you're not able to access uploaded files stored by PocketBase?

**Reply by MalikWhitten67** [reply]
yeah i downloaded the pb_data and added it locally but all files are not loaded.
![image](https://user-images.githubusercontent.com/65188863/219879504-cac17296-2b40-4c0b-b18f-2846fad312ab.png)

**Reply by benallfree** [reply]
So do you mean you have the files locally, but they are not showing up in the admin? Are you sure your local PocketBase is pointing to the correct static asset path?

**Comment by MalikWhitten67** [comment]
is there a way to get it or ill just have to start fresh

**Reply by benallfree** [reply]
See above, I'm not sure if I understand your question. Are you having trouble downloading all your assets, or do you have them locally but are having trouble accessing them from your local PocketBase?

**Reply by MalikWhitten67** [reply]
Each message has the ability to have a file. I was wondering why they are not working and showing on the messages. I ran a local instance and copied the database but was not able to see the files from each collection record

**Reply by benallfree** [reply]
When you say you copied the database, do you mean just the *.db files? Uploads are stored separately. I’m not super familiar with this feature of PB but I’ll check more if you let me know your subdomain.

**Reply by MalikWhitten67** [reply]
aww ok the subdomain is postr.pockethost.io

**Reply by MalikWhitten67** [reply]
yeah i copied every file under pb_data

**Reply by benallfree** [reply]
Ok and just to confirm, you see `pb_data/storage/**` in your local directory? I want to confirm that you actually downloaded the files.

**Reply by MalikWhitten67** [reply]
Yeah it was empty

**Reply by benallfree** [reply]
OK let me check into that. That may be a bug on my end.

**Comment by benallfree** [comment, marked as answer]
This is fixed in 0.7.1 and live now.
